README.md

```diff
@@ -50,6 +50,9 @@ For incremental builds, the `rush build` command can be used to only build packa
 4. Locally commit changes: `git commit` (or use the Visual Studio Code user interface)
 5. Repeat steps 1-4 until ready to push changes
 6. Check for API signature changes: `rush extract-api`. This will update the signature files, located in `common/api`.
+    - Be sure that your branch is up to date with `master` (i.e. `git merge origin/master`)
+    - Cleanup your cache: `rush clean`, `rush purge`
+    - Rebuild the project: `rush udpate`, `rush build`
```

Comment:
Do not encourage people to run `rush update` unless they've changed package.json files.

Reply:
hm... Shouldnt then we have all verstions set to specific ones (do not use `^` or `~` ?
but removed this command and left only `rush build`